Source/console.cpp

```diff
@@ -159,7 +159,7 @@ ConsoleTable::ConsoleTable(const std::optional<juce::String>& csv /*= std::nullo
     // Init CSV
     if (csv.has_value() && !csv->isEmpty())
     {
-        m_csvFile = juce::File::createLegalPathName(csv.value());
+        m_csvFile = juce::File::createLegalPathName(csv.operator*());
```

Comment:
I've never seen/used this manual call to the dereference operator for a std::optional. Why opt for this over .value()?

Reply:
.value() throws a `value' is unavailable: introduced in macOS 10.13` value error when compiling with XCode, afaik .operator*() works the same while avoiding the error

Reply:
Wow, that's wild. Would you mind replacing it with `*csv` then? The `.operator*()` syntax is pretty odd.